Expensify/App
# [No QA] Run E2E performance tests imperatively

### Details
This PR makes it so that we can manually trigger E2E performance tests on any given branch, commit, or tag.

Note: The main thing I'm unsure about with this PR is if we need to add any actor validation to prevent the e2e test workflow from being run by non-whitelisted actors.

### Fixed Issues
$ https://github.com/Expensify/App/issues/15605

### Tests
1. Merge this PR.
1. Verify that E2E performance tests run as expected.
1. Open the GH Actions console.
1. Manually trigger the new `e2ePerformanceTests` workflow with a specific branch.
1. Verify that the E2E performance tests run on that branch as expected.


### Offline tests
None

### QA Steps
None.


### PR Author Checklist

### Screenshots/Videos
No screenshots for this GitHub Actions change.
